docs/getting-started/server/guide.md

```diff
@@ -276,6 +276,14 @@ To run your local server environment as a licensed instance, you will need to do
 `Licensing Certificate - Dev` from the shared Engineering collection and install it. This can be
 done by double-clicking on the downloaded certificate.
 
+- Log in to your company-issued Bitwarden account
+- Navigate to the Bitwarden Vault
+- Scroll down to `Licensing Certificate \* Dev
```

Comment:
"Navigate to the Bitwarden Vault" is a bit redundant, that's where you land after logging in.

If you want this clarification, you could include it as part of the second point:

```
On the "Vaults" page, scroll down to the "Licensing Certificate - Dev" item
```

There are also some typos in the item name here. The above suggestion is optional, but please fix the typos either way.